src/lib/utils/committee-ordering/sort.ts

```diff
@@ -36,7 +36,7 @@ export function sortCommitteePos(
   posId2: Position["id"],
   committeeShortName: NonNullable<Committee["shortName"]>,
 ): number {
-  if (!(committeeShortName in committeeEnums)) return 1; // No committee? Assume pos B comes before
+  if (!(committeeShortName in committeeEnums)) return 0; // No committee? Preserve order
```

Comment:
I _think_ this makes more sense, but I'm not sure.

Reply:
Agreed!